presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/EnvironmentModule.java

```diff
@@ -49,12 +50,18 @@ public void configure(Binder binder)
         binder.bind(Kerberos.class);
 
         MapBinder<String, EnvironmentProvider> environments = newMapBinder(binder, String.class, EnvironmentProvider.class);
+
         environments.addBinding("singlenode").to(Singlenode.class);
+
         environments.addBinding("singlenode-hive-impersonation").to(SinglenodeHiveImpersonation.class);
+        environments.addBinding("singlenode-kerberos-hive-impersonation").to(SinglenodeKerberosHiveImpersonation.class);
+
         environments.addBinding("singlenode-hdfs-impersonation").to(SinglenodeHdfsImpersonation.class);
         environments.addBinding("singlenode-kerberos-hdfs-impersonation").to(SinglenodeKerberosHdfsImpersonation.class);
         environments.addBinding("singlenode-kerberos-hdfs-no-impersonation").to(SinglenodeKerberosHdfsNoImpersonation.class);
```

Comment:
I think we could try to merge metastore and hdfs impersonation into single environment and have single environment with no impersonation. Maybe  a follow up?

Reply:
Certainly a follow up. 
I don't want to introduce unnecessary changes during this refactoring.